docs/source/numpy.rst

```diff
@@ -347,11 +347,11 @@ hold any values and are computed upon access or assigmnent.
 +-----------------------------------------------+-----------------------------------------------+
 | ``np.sum(a)``                                 | ``xt::sum(a)``                                |
 +-----------------------------------------------+-----------------------------------------------+
-| ``np.prod(a, axis=1)``                        | ``xt::prod(a, {1})``                          |
+| ``np.prod(a, axis=1)``                        | ``xt::prod(a, 1)``                            |
```

Comment:
I think it could be nice to add a row where many axes are specified so the user knows she has to use initializer list syntax in that case.